tests/zerobase/enc/enc.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/tests.h>
+#include "zerobase_t.h"
+
+const char* ProtectedMessage = "Hello world from Enclave\n\0";
+
+int secure_str_patching(const char* src, char* dst, size_t dst_length)
```

Comment:
Please update per https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md#coding-conventions guidelines to avoid abbreviations (no "src", "dst", or "str")